narwhal/types/src/primary.rs

```diff
@@ -666,7 +681,7 @@ impl Certificate {
         // Verify the signatures
         let certificate_digest: Digest<{ crypto::DIGEST_LENGTH }> = Digest::from(self.digest());
         self.aggregated_signature
-            .verify(&pks[..], certificate_digest.as_ref())
+            .verify_secure(&to_intent_message(certificate_digest), &pks[..])
```

Comment:
Does Narwhal still use `verify()` in other callsites? And if yes, should they be switched over to `verify_secure()`?

Reply:
this is the only caller for verify, vote.verify and certificate.verify both use verify_secure as now